middlewares/authorization.js

```diff
@@ -0,0 +1,13 @@
+const jwt = require("jsonwebtoken");
+
+const authenticateUser = (req, res, next) => {
+  try {
+    const token = req.headers.authorization.split(" ")[1];
+    jwt.verify(token, process.env.SECRET_KEY);
+    next();
+  } catch (err) {
+    next(err);
+  }
+};
```

Comment:
```suggestion
const authenticateUser = (req, res, next) => {
  try {
    const token = req.headers.authorization.split(" ")[1];
    const { email } = jwt.verify(token, process.env.SECRET_KEY);
    res.locals.userEmail = email;
    next();
  } catch (err) {
    next(err);
  }
};
```
검증 후, 유저관련 정보를 여러 엔드포인트에서 사용하려면 이런 코드들이 필요할 것 같은데,
이후 제가 엔드포인트 만들면서 필요하면 추가하도록 할게요.